test/schema/GithubApi.Gist.Fetch.test.js

```diff
@@ -0,0 +1,45 @@
+const { expect } = require('chai');
+const axios = require('axios');
+const chaiSubset = require('chai-subset');
+const chai = require('chai');
+
+const body = {
+  description: 'Promise #1',
+  public: true,
+  files: {
+    'Readme.md': {
+      content: 'Promise #1'
+    }
+  }
+};
+
+chai.use(chaiSubset);
+
+const fetch = axios.create({
+  headers: {
+    Authorization: `token ${process.env.ACCESS_TOKEN}`
+  }
+});
```

Comment:
the idea of this point is to use the isomorphic-fetch library instead of axios.

Reply:
done